backend/server.py

```diff
@@ -0,0 +1,63 @@
+import db_interface
+from flask import Flask, jsonify, request, Response, json
+from werkzeug.exceptions import InternalServerError, BadRequest, HTTPException, NotFound
+
+app = Flask(__name__)
+PORT = 3003
+
+
+def configure_routes(app):
+    @app.route("/", methods=["GET"])
+    def hello():
+        """
+        Sample route
+        """
+        return "Hello World!"
+
+    @app.route("/get-tree-by-id", methods=["GET"])
+    def get_tree_by_id():
+        """
+        Expects the body of the request to have an id key; returns
+        the tree corresponding to an ID else {} if there is no such
+        tree.
+        """
+        body = request.json
+        if body is None or "id" not in body:
```

Comment:
The tree id should be part of the route url, instead of the request body. The route should probably be something like: "/trees/<int:id>"

Reply:
This makes the code a lot shorter, ty!